### PR TITLE
Boolean parameter for EU endpoint

### PR DESCRIPTION
**Pull request by Debora38**

### CODE REVIEW

**Comment by Debora38** [comment]
@rockem I'm wondering if this code with the boolean and an if case instead, is what you meant.

File: index.js [hunk 1]
```diff
@@ -1,9 +1,18 @@
...
+if (core.getInput('opsgenie_eu')) {
+    opsgenie.configure({
+        'api_key': core.getInput('api_key'),
+        'host': 'https://api.eu.opsgenie.com',
+    });
+} else {
+    opsgenie.configure({
+        'api_key': core.getInput('api_key'),
+    });
+}
```
Comment (rockem) [inline index.js:14]:
```suggestion
connectionDetails = {'api_key': core.getInput('api_key')}
if (core.getInput('opsgenie_eu')) {
    connectionDetails.set('host', 'https://api.eu.opsgenie.com')
 }
 opsgenie.configure(connectionDetails)
```

File: action.yml [hunk 2]
```diff
@@ -22,6 +22,10 @@ inputs:
   tags:
     description: Comma delimited list of tags
     required: false
+  opsgenie_eu:
```
Comment (rockem) [inline action.yml:25]: How about calling it 'using_eu_url'? So it will be clear it's a boolean

**Comment by rockem** [comment]
> @rockem I'm wondering if this code with the boolean and an if case instead, is what you meant.

Yes, pretty much

**Review by rockem — COMMENTED** [review]
Can you please update the README as well?

**Comment by Debora38** [comment]
> Can you please update the README as well?

@rockem ah yes, done

**Review by rockem — APPROVED** [review]

**Pull request merged**